src/main/java/gov/loc/mets/Helper.java

```diff
@@ -45,6 +35,15 @@
 import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 
+import gov.loc.mets.DivType.Fptr;
```

Comment:
Why are you moving the gov import statements to the end of import statements? Without your changes import statements was sorted by alphabet.

Reply:
Having import statements in alphabetical order with empty lines between the blocks requires some manual configuration in Eclipse. All package groups must be configured manually, all other packages go below there. When I set this up, I did a grep for `^import ([^.]+)` on all the kitodo-production *.java files, and added the found package groups in alphabetical order. However, the package starting with `gov` was not part of kitodo-production so I did not add it. I have now updated the configuration.